index.html

```diff
@@ -144,7 +145,27 @@ <h4>LET’S CONNECT</h4>
         </ul>
     </div>
  </section>
+ 
+</div>
+</div>
+<!--------------------------------contact form ----------------------------------->
+<section class="contact">
+    <div class="contact_width">
+            <p>Contact me</p>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form action="https://formspree.io/f/xlekbwnv" method="POST"> 
+        <label for="name">Name</label>
+            <input name="text" id="name" type="text" placeholder="your name"  maxlength="30">
+            <br>
+            <label for="email">Email</label>
+            <input name="email" id="email" type="email" placeholder="your Email" maxlength="30">
+            <br>
+           <label for="message"></label>
+           <textarea name="message-box" id="message" cols="30" rows="10" maxlength="500" placeholder="Write your message here" type="text"></textarea>
+           <button type="submit">Get in touch</button>
+        </form>
```

Comment:
- [ ] Kindly make sure that user can't submit the form if there is one or more of the fields is empty as per this [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#:~:text=A%20user%20cannot%20submit%20the%20form%20if%20one%20or%20more%20of%20the%20fields%20is%20empty.). Currently, I can submit the form without entering any values.